Library/Homebrew/cmd/audit.rb

```diff
@@ -367,6 +367,10 @@ def audit_desc
     if desc =~ %r[[Cc]ommand line]
       problem "It should be \"command-line\", not \"command line\"."
     end
+
+    if desc =~ %r[^([Aa]n?)\s]
+      problem "Description begins with indefinite article: \"#{$1}\""
```

Comment:
Might want to clarify `(i.e. 'a' or 'an')` just for people who don't know what an indefinite article is.